<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPage</class>
 <widget class="QWidget" name="SettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="tag1"/>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="file1"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="file2"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="file4"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="tag2"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="tag3"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="tag4"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="file3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>files</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>tags</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="start">
       <property name="text">
        <string>start </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
